src/com/company/factory/MatrixOperation.java

```diff
@@ -0,0 +1,37 @@
+package com.company.factory;
+
+import com.company.model.IntegerMatrix;
+
+public class MatrixOperation {
+
+    IntegerMatrix matrix;
+
+    public void multiplyMatrix(int[][] arr1, int[][] arr2) {
```

Comment:
multiply should return the matrix object and you should print it's data in the above layer